README.md

```diff
@@ -20,6 +20,25 @@ Contributing to FoundationDB can be in contributions to the code base, sharing y
 
 ## Getting Started
 
+### Latest Stable Releases
+
+The lastest stable releases are (were) versions that are recommended for production use, which have been extensively validated via simulation and real cluster tests and used in our production environment.
+
+| Branch   |     Latest Production Release      |  Notes |
+|:--------:|:-------------:|------:|
+| 7.3      |  [7.3.43](https://github.com/apple/foundationdb/releases/tag/7.3.43) | Supported |
+| 7.2      |   | Experimental branch |
+| 7.1      |  [7.1.57](https://github.com/apple/foundationdb/releases/tag/7.1.57)   |   Bug fixes |
+| 7.0      |           |   Experimental branch |
+| 6.3      |  [6.3.25](https://github.com/apple/foundationdb/releases/tag/6.3.25) |   Unsupported |
+
+- Supported branches are those we actively maintain and will publish new patch releases.
+- Bug fixes are branches we still accept bug fixes, but may not publish new patch releases.
```

Comment:
```suggestion
- `Bug fixes` are branches for which we still accept bug fixes, but they may not receive any new features.
```

Also, what is meant by "but may not publish new patch releases"? Does that mean that we are not going to patch the bugs, or we are not deploying new features? We may want to be more explicit on what to expect for future patch releases.

Reply:
We'll patch the bug in the branch's code, but we may not create newer patch releases. I.e., community can build their own release binaries if needed.